ui/components/multichain/token-list-item/token-list-item.js

```diff
@@ -153,15 +184,21 @@ export const TokenListItem = ({
                 </Text>
               )}
             </Box>
-            <Text
-              fontWeight={FontWeight.Medium}
-              variant={TextVariant.bodyMd}
-              width={BlockSize.TwoThirds}
-              textAlign={TextAlign.End}
-              data-testid="multichain-token-list-item-secondary-value"
-            >
-              {secondary}
-            </Text>
+            {showScamWarning ? (
+              <ButtonIcon
+                iconName={IconName.Danger}
+                onClick={(e) => {
+                  e.preventDefault();
+                  e.stopPropagation();
+                  setShowScamWarningModal(true);
+                }}
+                color={IconColor.errorDefault}
+                size={IconSize.Lg}
+                backgroundColor={BackgroundColor.transparent}
+              />
+            ) : (
+              secondary
+            )}
```

Comment:
Nit: I think it should be inside the Text as it was previously 
```suggestion
            ) : (
              <Text
              fontWeight={FontWeight.Medium}
              variant={TextVariant.bodyMd}
              width={BlockSize.TwoThirds}
              textAlign={TextAlign.End}
              data-testid="multichain-token-list-item-secondary-value"
            >
              {secondary}
            </Text>
            )}
```

Reply:
Wow, good shout!  Phew!